profile/README.md

```diff
@@ -14,32 +14,29 @@
 Miden is the edge blockchain that grants applications the power to scale with public and private transactions. It aims at builders who want to create an ecosystem for private asset management and OTC-trading.
 
 ## Project structure
-This repository stores all the code for Polygon Miden. Everything is open source. 
+This repository stores all the code for Miden. Everything is open source. 
 
 | Repository              | Description |
 | ----------------------- | ----------- |
-| [Miden Base](https://github.com/0xMiden/miden-base)    | Core components of the Polygon Miden rollup. |
+| [Miden Base](https://github.com/0xMiden/miden-base)    | Core components of the Miden chain. |
```

Comment:
We should probably say "blockchain" instead of just "chain" here and in other places.